Dockerfile

```diff
@@ -32,12 +34,16 @@ RUN go test .
 
 
 # Create production image for running node feature discovery
-FROM debian:stretch-slim
+FROM BASEIMAGE2
```

Comment:
Similar to the build stage, would it be possible to use ARG here?

Reply:
Done.